src/absil/ilread.fs

```diff
@@ -3970,15 +3970,16 @@ let ClosePdbReader pdb =
 #endif
 
 let OpenILModuleReader infile opts = 
-    if IL.runningOnWindows then
```

Comment:
i found about that searching for perfomance improvement. 
because the function are not inlined, are called a lot (really, like getByte), and is a bit of tax. 
i think if we can inline that's going away too, but i found really hard to do that because there was also binaryfile.
maybe i need to continue that old branch sometimes. or someone from community can volounteer (it's going to help a lot in .net core and mono)